### PR TITLE
Widget Docs

### PR DESCRIPTION
**Pull request by cs-rohit-khune**

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 | [Home](../README.md) |
-|--------------------------------------------|
+|---------------------------------------------|
 
 # Usage
```
Comment (cs-abhinav-goel) [inline docs/usage.md:5]: Update the below line
>Line#10 >> Rephrase the sentence as per the new Label given to the field
>Update the image "custom-module.png"
>Update image "filter-for-single-module.png"
>Crop the Image "custom-module-layer.png" and remove the "Add Layer" button from bottom
>Rephrase the line #41 >> Avoid using the term "Custom module"
>In line#54, change the value "0" to "?" >> Also remove " (In this case for incident) "
>Line#61 > Refrain using the term "Multiple Modules" and use "Count of Records Across Module"

Comment (cs-abhinav-goel) [reply docs/usage.md:5]: Also Update the image "funnel.png"
image

File: docs/usage.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 
 **Funnel Chart Widget Edit View**
 
 **1. Data Source : Record containing JSON Data**
 
-Select the "Single Record" option if all the data to be rendered are in a single record of a certain field of the module.
+Select the "Record containing JSON Data" option if all the data to be rendered sing JSON data of a certain field value of the module.
```
Comment (cs-suyog-jinturkar) [inline docs/usage.md]: sing??

Comment (cs-rohit-khune) [reply docs/usage.md]: corrected the typo.

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**